src/metrics/family.rs

```diff
@@ -234,9 +234,35 @@ impl<S: Clone + std::hash::Hash + Eq, M, C: MetricConstructor<M>> Family<S, M, C
         })
     }
 
+    /// Remove a label set from the metric family.
+    ///
+    /// Returns a bool indicating if a label set was removed or not.
+    ///
+    /// ```
+    /// # use prometheus_client::metrics::counter::{Atomic, Counter};
+    /// # use prometheus_client::metrics::family::Family;
+    /// #
+    /// let family = Family::<Vec<(String, String)>, Counter>::default();
+    ///
+    /// // Will create the metric with label `method="GET"` on first call and
+    /// // return a reference.
+    /// family.get_or_create(&vec![("method".to_owned(), "GET".to_owned())]).inc();
+    ///
+    /// // Will return `true`, indicating that the `method="GET"` label set was
+    /// // removed.
+    /// assert!(family.remove_label_set(&vec![("method".to_owned(), "GET".to_owned())]));
+    /// ```
+    pub fn remove_label_set(&self, label_set: &S) -> bool {
+        self.write().remove(label_set).is_some()
+    }
+
     pub(crate) fn read(&self) -> RwLockReadGuard<HashMap<S, M>> {
         self.metrics.read()
     }
+
+    fn write(&self) -> RwLockWriteGuard<HashMap<S, M>> {
+        self.metrics.write()
+    }
```

Comment:
I am guessing you added this method for the sake of consistency with the `read` method above? I don't think it is necessary. The `read` method only exists as code within the `crate` but outside this `mod` needs it.

Reply:
Yeah. This was for consistency. Im happy to remove it if needed.